app/script/auth/Runtime.js

```diff
@@ -55,15 +57,25 @@ export default class Runtime {
   getBrowserName = () => this.getPlatform().name.toLowerCase();
   getBrowserVersion = () => this.getPlatform().version;
   getMajorBrowserVersion = () => parseInt(this.getPlatform().version.split('.')[0]);
+  getMinorBrowserVersion = () => parseInt(this.getPlatform().version.split('.')[1]);
```

Comment:
Let's not do the version split twice but do a get browser version thingy that returns all parts of the version